lib/utils/report.cjs

```diff
@@ -168,16 +168,16 @@ function isFixApplied({ fix, line, result: { stylelint }, ruleName }) {
 		return false;
 	}
 
-	const hasFix = Boolean(config.fix && !config.rules?.[ruleName][1]?.disableFix);
-	const fixed = hasFix && (config.ignoreDisables || !isDisabled(ruleName, line, disabledRanges));
+	const shouldFix = Boolean(config.fix && !config.rules?.[ruleName][1]?.disableFix);
```

Comment:
`hasFix` is easily confused with "has a fixer callback".
While these conditions check the config to make sure that fixes **should** be applied when possible.